kcsd/KCSD.py

```diff
@@ -1066,6 +1065,126 @@ def int_pot_3D(self, xp, yp, zp, x, R, h, basis_func):
         pot *= basis_func(dist, R)
         return pot
 
+class oKCSD2D(KCSD2D):
+    """oKCSD - The variant for the Kernel Current Source Density method that 
+    allows to reconstruct potential and CSD in given 2D space points.
+    """
+    def __init__(self, ele_pos, pots, **kwargs):
+        """Initialize oKCSD2D Class.
+        Parameters
+        ----------
+        ele_pos : numpy array
+            positions of electrodes
+        pots : numpy array
+            potentials measured by electrodes
+        **kwargs
+            configuration parameters, that may contain the following keys:
+            src_type : str
+                basis function type ('gauss', 'step', 'gauss_lim')
+                Defaults to 'gauss'
+            sigma : float
+                space conductance of the tissue in S/m
+                Defaults to 1 S/m
+            n_src_init : int
+                requested number of sources
+                Defaults to 1000
+            R_init : float
+                demanded thickness of the basis element
+                Defaults to 0.23
+            h : float
+                thickness of analyzed tissue slice
+                Defaults to 1.
+            own_est: numpy array
+                points coordinates of the basis source centers 
+                and estimation places
+            lambd : float
+                regularization parameter for ridge regression
+                Defaults to 0.
+        Raises
+        ------
+        LinAlgError
+            Could not invert the matrix, try changing the ele_pos slightly
+        KeyError
+            Basis function (src_type) not implemented.
+            See basis_functions.py for available
+        """
+        super().__init__(ele_pos, pots, **kwargs)
```

Comment:
super(oKCSD2D, self).__init__(ele_pos, pots, **kwargs)

Reply:
There is a nice feature with the "+- on a stack of papers" widget:
```suggestion
        super(oKCSD2D, self).__init__(ele_pos, pots, **kwargs)
```